tests/std/tests/P0009R18_mdspan_mdspan_death/test.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _CONTAINER_DEBUG_LEVEL 1
+
+#include <array>
+#include <mdspan>
+
+#include <test_death.hpp>
+
+using namespace std;
+
+constexpr array<int, 256> some_ints{};
+
+void test_construction_from_other_mdspan() {
+    auto mds1 = mdspan{some_ints.data(), 8, 2, 8};

```

Comment:
This is an occurrence of "almost always auto" which we conventionally avoid. `mdspan mds1{some_ints.data(), 8, 2, 8};` is the preferred form.

Occurs below.